packages/itwin/tree-widget/src/test/trees/models-tree/internal/ModelsTreeVisibilityHandler.test.ts

```diff
@@ -168,7 +169,16 @@ describe("ModelsTreeVisibilityHandler", () => {
           imodelAccess: createFakeIModelAccess(),
         });
 
-        const node = createElementHierarchyNode({ modelId: "0x1", categoryId: "0x2", elementId: "0x3" });
+        const node = createElementHierarchyNode({
+          modelId: "0x1",
+          categoryId: "0x2",
+          elementId: "0x3",
+          parentKeys: [
+            { type: "instances", instanceKeys: [{ id: "0x1", className: MODEL_CLASS_NAME }] },
```

Comment:
it's very unlikely that the hierarchy will start with a Model node - could these `parentKeys` be adjusted to have at least one `bis.Subject` key in front?